.github/workflows/build-tool-windows.yml

```diff
@@ -1,10 +1,16 @@
-name: Build-Tools
+name: Build-Tools-msys2-Windows
 
-on: [push, pull_request]
+on:
+  push:
+    paths:
+      - 'tools/*'
+  pull_request:
+    paths:
+      - 'tools/*'
```

Comment:
this is wrong because tools include also files from src

Reply:
reverted.